client/scripts/rails_only.jsx

```diff
@@ -8,3 +8,7 @@ require('es5-shim/es5-sham');
 // Due to issue: https://github.com/ariya/phantomjs/issues/12401
 // Phantomjs does not like promises
 require('es6-promise').polyfill();
+
+require('expose?jQuery!jquery');
```

Comment:
This should stay in the client/webpack.rails.config.js. No reason to put it here, is there?